tests/io/CopyTest.php

```diff
@@ -0,0 +1,45 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+use namespace HH\Lib\IO;
+use namespace HH\Lib\File;
+
+use function Facebook\FBExpect\expect; // @oss-enable
+use type Facebook\HackTest\HackTest; // @oss-enable
+// @oss-disable: use type HackTest;
+
+// @oss-disable: <<Oncalls('hack')>>
+final class CopyTest extends HackTest {
+  public async function testCopy(): Awaitable<void> {
+    await using $source = File\temporary_file();
+    await using $target = File\temporary_file();
+
+    await $source->writeAsync('Hello, World!');
+    await $source->seekAsync(0);
+
+    expect($source->isEndOfFile())->toBeFalse();
+    expect($target->isEndOfFile())->toBeFalse();
```

Comment:
Are you testing it a newly opened file is not eof?
Is that part of the `copy()`?

Reply:
somehow, since after `copy` it should be at the eof.

its okay to test extra features of `IO` here and there since there's no tests for these methods anywhere else.